Vic2ToHoI4/Source/HOI4World/HoI4Country.h

```diff
@@ -69,6 +70,13 @@ class Country
 		 const Regions& regions,
 		 Mappers::GraphicsMapper& graphicsMapper,
 		 Names& names);
+	explicit Country(const std::string& tag_,
```

Comment:
Add a comment describing what case this constructor is for, as I've recently added for the others: https://github.com/ParadoxGameConverters/Vic2ToHoI4/blob/master/Vic2ToHoI4/Source/HOI4World/HoI4Country.h